src/style.css

```diff
@@ -1 +1,34 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 20px;
+}
+
+.stars:last-of-type {
+  height: 16px;
+}
```

Comment:
you don't need the vertical gap in this task

Reply:
<img width="425" alt="image" src="https://github.com/user-attachments/assets/fc8494e9-ad22-47bf-a7bf-75625aa90d99">

It looks like we need the gap